zabbix_auto_config/processing.py

```diff
@@ -225,7 +225,11 @@ def __init__(self, name, state, db_uri, host_modifier_dir):
     def get_host_modifiers(self):
         sys.path.append(self.host_modifier_dir)
 
-        module_names = [filename[:-3] for filename in os.listdir(self.host_modifier_dir) if filename.endswith(".py")]
+        try:
+            module_names = [filename[:-3] for filename in os.listdir(self.host_modifier_dir) if filename.endswith(".py")]
+        except(FileNotFoundError):
+            logging.error(f"Host modififier directory {host_modifier_dir} does not exist.")
```

Comment:
Formatting should generally be passed as parameters. [W1203](https://pylint.pycqa.org/en/latest/messages/warning/logging-fstring-interpolation.html?highlight=W1203).

Reply:
Thanks for this pointer, I had no idea :)